src/lando/api/legacy/workers/landing_worker.py

```diff
@@ -163,6 +163,19 @@ def run_job(self, job: LandingJob) -> bool:
 
         return True
 
+    def convert_patches_to_diff(self, scm: AbstractSCM, job: LandingJob):
+        """Generate a unified diff from multiple patches stored in a revision."""
+        # NOTE: this only applies to git patches that are downloaded from GitHub
+        # at this time. In theory this would work for any provided patches in a
+        # standard format.
+
+        for revision in job.revisions:
+            if not revision.patches:
+                continue
+            diff = scm.get_diff_from_patches(revision.patches)
```

Comment:
`get_diff_from_patches` doesn't commit the changes, and only adds to the index.

I think this means that the patches for each revision are going to get larger and larger, with the old changes from all the previous revisions.

I guess we don't see that at the moment because we only deal with one revision per pull-request, and only a single pull-request per landing.